src/qtiItem/helper/maxScore.js

```diff
@@ -822,5 +822,21 @@ export default {
             max = 0;
         }
         return max;
+    },
+    pciInteraction: function pciInteraction(interaction) {
```

Comment:
```suggestion
    pciInteraction(interaction) {
```

And you can use const. This file will can be es6